### PR TITLE
Correct indentation in RBAC manifest yaml

### PR DESCRIPTION
**Pull request by adamancini**

### CODE REVIEW

**Comment by jonquil2002** [comment]
@adamancini The formatting is off in the published output. Some of it is falling outside of the code block. Please see preview: https://deploy-preview-1252--replicated-docs.netlify.app/enterprise/installing-general-requirements#rbac-requirements

I think it might be easier if you fix it in your branch bc you can print a local preview using `yarn start`

File: docs/enterprise/installing-general-requirements.mdx [hunk 1]
```diff
@@ -104,43 +104,44 @@ To install or upgrade KOTS with namespace-scoped access, the user must have _one
...
+   ```yaml
+   apiVersion: rbac.authorization.k8s.io/v1
+   kind: Role
+   metadata:
+     labels:
+       kots.io/backup: velero
+       kots.io/kotsadm: "true"
+     name: kotsadm-role
+   rules:
+   - apiGroups: [""]
```
Comment (jonquil2002) [inline docs/enterprise/installing-general-requirements.mdx:116]: @adamancini Shouldn't all of these apiGroups be indented? They are not publishing properly in code block - you can see it's messed up in the preview. And the previous example shows apiGroup as indented, so it's inconsistent.

Please fix and ping me when you're ready for another review. Thanks.

Comment (adamancini) [reply docs/enterprise/installing-general-requirements.mdx:116]: @jonquil2002 the yaml spec says that lists can be at the same level or indented 1 column, as long as it's consistent throughout.  When I was looking at the editor in GitHub, the previous paragraphs used the no-indent style.  Let me take another look and I'll fix it up.

**Comment by adamancini** [comment]
> @adamancini The formatting is off in the published output. Some of it is falling outside of the code block. Please see preview: https://deploy-preview-1252--replicated-docs.netlify.app/enterprise/installing-general-requirements#rbac-requirements
> 
> I think it might be easier if you fix it in your branch bc you can print a local preview using `yarn start`

To get that to work, do I need to do anything other than install `yarn`?


```
 alice  ~/replicated/replicated-docs   main  brew install yarn
Running `brew update --auto-update`...
==> Auto-updated Homebrew!
Updated 3 taps (homebrew/cask-fonts, homebrew/core and homebrew/cask).
==> New Formulae
blink                 couchbase-shell       go-feature-flag       pixi                  ruff-lsp
botan@2               espflash              neonctl               pylyzer               woof-doom
==> New Casks
ente                                 maa                                  poe

You have 6 outdated formulae and 1 outdated cask installed.

==> Fetching yarn
==> Downloading https://ghcr.io/v2/homebrew/core/yarn/manifests/1.22.19
######################################################################################################## 100.0%
==> Downloading https://ghcr.io/v2/homebrew/core/yarn/blobs/sha256:79b90324a5365189a144b786e9bdb3bf32be3823e904
######################################################################################################## 100.0%
==> Pouring yarn--1.22.19.all.bottle.tar.gz
==> Caveats
yarn requires a Node installation to function. You can install one with:
  brew install node
==> Summary
🍺  /opt/homebrew/Cellar/yarn/1.22.19: 15 files, 5MB
==> Running `brew cleanup yarn`...
Disable this behaviour by setting HOMEBREW_NO_INSTALL_CLEANUP.
Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).
 ✔  alice  ~/replicated/replicated-docs   main  yarn start
yarn run v1.22.19
warning ../../package.json: No license field
$ docusaurus start
/bin/sh: docusaurus: command not found
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by paigecalvert** [comment]
> To get that to work, do I need to do anything other than install `yarn`?

@adamancini I believe that you would also need to run `yarn add docusaurus`

**Comment by adamancini** [comment]
> > To get that to work, do I need to do anything other than install `yarn`?
> 
> @adamancini I believe that you would also need to run `yarn add docusaurus`

thanks! I see it in the contributing docs now :|

**Comment by paigecalvert** [comment]
> thanks! I see it in the contributing docs now :|

no worries! I actually tried to look for it in our wiki and couldn't find it. Lol. So you are one step ahead of me there

**Comment by adamancini** [comment]
@paigecalvert @jonquil2002 should be ready for re-review now

**Review by jonquil2002 — APPROVED** [review]
lgtm - @adamancini Thank you so much!

**Pull request merged**